srml/bridge/src/lib.rs

```diff
@@ -160,6 +161,39 @@ impl<T: Trait> Module<T> {
 	}
 }
 
+// A naive way to check whether a `child` header is an ancestor
+// of an `ancestor` header. For this it requires a proof which
+// is a header chain, This could be updated to use something like
+// Log2 Ancestors (#2053) in the future.
+fn verify_ancestry<H>(proof: Vec<H>, ancestor: H, child: H) -> std::result::Result<(), Error>
```

Comment:
nit: Why does this need to be a `std::result::Result` and not just `Result`?

Reply:
In another case I was having an issue where it was using a Substrate `Result` instead of an `std::Result` which is what I wanted. Looks like the import is gone now though.